src/wallet/wallet2.cpp

```diff
@@ -8632,16 +8638,21 @@ uint64_t wallet2::adjust_mixin(uint64_t mixin)
   return mixin;
 }
 //----------------------------------------------------------------------------------------------------
-uint32_t wallet2::adjust_priority(uint32_t priority)
+FeePriority wallet2::adjust_priority(uint32_t priority)
+{
+  return adjust_priority(FeePriorityUtilities::FromIntegral(priority));
+}
+//----------------------------------------------------------------------------------------------------
+FeePriority wallet2::adjust_priority(FeePriority priority)
 {
-  if (priority == 0 && m_default_priority == 0 && auto_low_priority())
+  if (priority == FeePriority::Default && m_default_priority == FeePriority::Default && auto_low_priority())
   {
     try
     {
       // check if there's a backlog in the tx pool
       const bool use_per_byte_fee = use_fork_rules(HF_VERSION_PER_BYTE_FEE, 0);
-      const uint64_t base_fee = get_base_fee(1);
-      const double fee_level = base_fee * (use_per_byte_fee ? 1 : (12/(double)13 / (double)1024));
+      const uint64_t base_fee = get_base_fee(FeePriority::Unimportant);
+      const double fee_level = base_fee * (use_per_byte_fee ? 1 : (12. / 13. / 1024.));
```

Comment:
The `double` changes seems unrelated and unnecessary (revert).

Reply:
Done